qrisk/tests/test_stats.py

```diff
@@ -127,6 +168,43 @@ def test_max_drawdown(self, returns, expected):
             expected,
             DECIMAL_PLACES)
 
+    # Multiplying returns by a positive constant larger than 1 will increase
+    # the maximum drawdown by a factor greater than or equal to the constant.
+    # Similarly, a positive constant smaller than 1 will decrease maximum
+    # drawdown by at least the constant.
+    @parameterized.expand([
+        (noise_uniform, 1.1),
+        (noise, 2),
+        (noise_uniform, 10),
+        (noise_uniform, 0.99),
+        (noise, 0.5)
+    ])
+    def test_max_drawdown_transformation(self, returns, constant):
+        max_dd = qrisk.max_drawdown(returns)
+        transformed_dd = qrisk.max_drawdown(constant*returns)
+        if constant >= 1:
+            assert constant*max_dd <= transformed_dd
+        else:
+            assert constant*max_dd >= transformed_dd
+
+    # Translating returns by a positive constant should increase the maximum
+    # drawdown to a maximum of zero. Translating by a negative constant
+    # decreases the maximum drawdown.
+    @parameterized.expand([
+        (noise, .0001),
+        (noise, .001),
+        (noise_uniform, .01),
+        (noise_uniform, .1),
+    ])
+    def test_max_drawdown_translation(self, returns, constant):
+        depressed_returns = returns-constant
+        raised_returns = returns+constant
+        max_dd = qrisk.max_drawdown(returns)
+        depressed_dd = qrisk.max_drawdown(depressed_returns)
+        raised_dd = qrisk.max_drawdown(raised_returns)
+        assert max_dd <= raised_dd
+        assert depressed_dd <= max_dd
```

Comment:
The test translates returns up and down by adding and subtracting to the constant. A test for negative constants would change the assertions at the end.